parity/url.rs

```diff
@@ -16,34 +16,19 @@
 
 //! Cross-platform open url in default browser
 
-#[cfg(windows)]
-mod shell {
-	extern crate winapi;
-
-	use self::winapi::*;
-	extern "system" {
-		pub fn ShellExecuteA(
-			hwnd: HWND, lpOperation: LPCSTR, lpFile: LPCSTR, lpParameters: LPCSTR, lpDirectory: LPCSTR,
-			nShowCmd: c_int
-		) -> HINSTANCE;
-	}
-
-	pub use self::winapi::SW_SHOWNORMAL as Normal;
-}
-
 #[cfg(windows)]
 pub fn open(url: &str) {
 	use std::ffi::CString;
 	use std::ptr;
-
-	unsafe {
-		shell::ShellExecuteA(ptr::null_mut(),
-			CString::new("open").unwrap().as_ptr(),
-			CString::new(url.to_owned().replace("\n", "%0A")).unwrap().as_ptr(),
-			ptr::null(),
-			ptr::null(),
-			shell::Normal);
-	}
+	use winapi::um::shellapi::ShellExecuteA;
+	use winapi::um::winuser::SW_SHOWNORMAL as Normal;
+
+	ShellExecuteA(ptr::null_mut(),
```

Comment:
Sorry, but I will repeat myself again because my comment was removed in the latest change!

But, actually `ShellExecuteA` can fail. Why doesn't we care about that?

https://msdn.microsoft.com/en-us/library/windows/desktop/bb762153(v=vs.85).aspx

Reply:
cause we're lazy :p created an issue for that :)